src/afl-cc.c

```diff
@@ -914,6 +918,13 @@ static void edit_params(u32 argc, char **argv, char **envp) {
 
   }
 
+  if (getenv("AFL_USE_LSAN")) {
+    cc_params[cc_par_cnt++] = "-includesanitizer/lsan_interface.h";
```

Comment:
is this line correct? it looks .... weird :)

Reply:
Yup it's correct, unless there's some other way afl does includes. To use __lsan_do_leak_check, the program needs to be built with sanitizer/lsan_interface.h

Reply:
I have never seen the option before :)

Reply:
I'm not sure how portable it is, but it should be supported by any compiler (aka just clang) which supports LSAN, anyways.